nwb_conversion_tools/basedatainterface.py

```diff
@@ -1,7 +1,7 @@
 """Authors: Cody Baker and Ben Dichter."""
 from abc import abstractmethod, ABC
 
-from .utils.json_schema import get_base_schema, get_schema_from_method_signature
+from .utils import get_base_schema, get_schema_from_method_signature
```

Comment:
What is the purpose of this change?

Reply:
Much easier to import utils when you don't have to think about which sub-module they belong to. Also prevents future additions/changes from breaking anything; tbh we probably always should have done it that way.